content/_posts/2022-04-14-la-communaute-na-pas-failli.md

```diff
@@ -0,0 +1,95 @@
+
+Si l’excès de sécurité est contre productif, après les 6 premiers mois il est utile de faire une première analyse des risques afin de cartographier les points de faiblesses les plus importants et la manière d’y répondre.
+
+Ensuite, avec le développement de la startup il s’agit de passer l’homologation de sécurité[homologations de sécurité](https://doc.incubateur.net/communaute/gerer-sa-startup-detat-ou-de-territoires-au-quotidien/je-securise-mon-produit/foire-aux-questions-sur-la-mise-en-conformite-des-services-numeriques/securite-and-homologation) obligatoires pour chaque produit en ligne. Elles sont un moyen d'inventorier l'écosystème et composants vulnérables (metabase, etc) d'un produit. Elles permettent enfin de s'assurer proportionnellement aux risques de l'adéquation des mesures (techniques ou organisationnelles) mises en place au sein de la startup d'État: outils automatisés d'alerte et de patchs ([Dependabot](https://github.com/dependabot) par exemple), dashlord, etc. Elles sont une photo des vulnérabilités et de leurs solutions à un instant T. En ce sens, elles doivent être renouvelées et sont complémentaires d’un dashlord automatiquement ou régulièrement mis à jour.  
+
+Les équipes support des incubateurs aident et appuient les startups d'État dans la mise en œuvre de ces démarches de prises en compte et d'amélioration continue de la sécurité de leurs produits. 
+
+#### Bonnes pratiques et ressources utiles à retenir 
+##### Bonnes pratiques
+Avoir le contact - mail et numéro de téléphone - de l’ensemble de l’équipe et en particulier des développeurs.
+
+En cas de faille, regarder si son code est touché et ses logiciels tiers. S’il y a un potentiel, suspendre le service. Vérifier ensuite entre le moment de la découverte et la correction qu’aucune exploitation n’ai eu lieu.
+
+Mettre en commun les informations sur les services touchés sur un tableau + communication via un canal dédié.
+
+Mobiliser l’ensemble de la communauté (dev mais aussi pm, design ou déploiement) pour s’assurer que toutes les Startup ont lancé une analyse.
+
+Avoir toujours un développeur disponible dans l’équipe pour suspendre le service en cas d’urgence.
+
+Réaliser une analyse des risques puis une homologation de sécurité.
+
+Développer un dashlord de suivi automatisé de la sécurité.
+
+
+
+##### Ressources utiles
+
+
+Un socle de services communs apporté par la communauté a d'ores et déjà montré son utilité dans cet épisode: 
+
+- [mattermost](https://mattermost.incubateur.net): communication rapide et simple pour alerter et s'entraider. Un canal dédié avait été créé. 
+- [offre incubateur d'hébergement PaaS Scalingo](https://scalingo.com): les équipes se concentrent sur leur app et l'hébergeur assure la sécurité de l'infra, 
+- [offre incubateur de WAF Baleen](https://baleen.com): filtre les requêtes réseau pour protéger encore plus les produits, notamment des attaques DdoS ou d’exploits comme celles de log4shell, 

```

Comment:
On a pas vraiment d'offre nous, vous si ?

Reply:
nous on a une offre scalingo
pour le reste do not know, @tristanrobert ?

Reply:
offre scalingo chez nous pour ceux qui le souhaitent, mais pas valable pour tous les incubateurs en effet